src/main/groovy/edu/oregonstate/mist/slashcourses/db/SlashCourseDAO.groovy

```diff
@@ -0,0 +1,26 @@
+package edu.oregonstate.mist.slashcourses.db
+
+import edu.oregonstate.mist.slashcourses.core.SlashCourse
+import edu.oregonstate.mist.slashcourses.mapper.SlashCourseMapper
+import org.skife.jdbi.v2.sqlobject.Bind
+import org.skife.jdbi.v2.sqlobject.SqlQuery
+import org.skife.jdbi.v2.sqlobject.customizers.RegisterMapper
+
+/**
+ * Slash Course DAO
+ */
+@RegisterMapper(SlashCourseMapper)
+public interface SlashCourseDAO {
+
+    /** Get slash course by CRN
+     *
+     *  @param crn
+     *  @return Slash Course object
+     */
+    @SqlQuery("""
+              SELECT *
+              FROM COURSE
+              WHERE CRN = :crn
+              """ )
+    SlashCourse getByCRN(@Bind("crn") Integer crn)
+}
```

Comment:
Both of your DAOs need an override close method. See: https://github.com/osu-mist/locations-api/blob/develop/src/main/groovy/edu/oregonstate/mist/locations/db/CampusMapLocationDAO.groovy#L67